compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/plugins/CompilerPlugins.java

```diff
@@ -164,14 +162,7 @@ public static List<String> annotationsAsStr(NodeList<AnnotationNode> supportedAn
     }
 
     public static String getUri(Node clientNode) {
-        BasicLiteralNode clientUri;
-
-        if (clientNode.kind() == SyntaxKind.MODULE_CLIENT_DECLARATION) {
-            clientUri = ((ModuleClientDeclarationNode) clientNode).clientUri();
-        } else {
-            clientUri = ((ClientDeclarationNode) clientNode).clientUri();
-        }
-
+        BasicLiteralNode clientUri = ((ModuleClientDeclarationNode) clientNode).clientUri();
```

Comment:
Do we need to cast this anymore ? We can change the line 166 as well.

Reply:
Sent a separate PR with some changes https://github.com/MaryamZi/ballerina/pull/16. Separate PR because this changes public APIs also.

Reply:
Verified the changes with @azinneera and pushed directly (8f97c47)